water-alternatives.csl

```diff
@@ -45,19 +45,31 @@
           </names>
           <group delimiter=" ">
             <choose>
-              <if variable="ISBN ISSN" match="any">
+              <if variable="ISBN ISSN container-title" match="any">
```

Comment:
it's seem like you want to add event to this list or am I misunderstanding this?